src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -43,10 +47,24 @@ public static PreSubmitCallbackResponse<SscsCaseData> processPostponementRequest
             return response;
         }
 
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        List<Hearing> allHearings = sscsCaseData.getHearings();
+        Hearing latestHearing =  allHearings.isEmpty() ? null : allHearings.get(allHearings.size() - 1);
```

Comment:
You need test coverage here.

Reply:
Ignore that. There's actually a getLatestHearing method on sscsCaseData

Reply:
There is a problem with using that method. When it tries to sort the hearing in descending order an UnsupportedOperationException is thrown

Reply:
Where does the error occur?

Reply:
Fixed it. When creating the test data, the list of hearings were immutable. So when it tries to get the latest hearing by sorting it in the reverse order, its not possible. Now I've changed the list to be mutable.